bvm/ballerina-logging/src/main/java/org/ballerinalang/logging/BLogManager.java

```diff
@@ -277,4 +278,20 @@ private InputStream propertiesToInputStream(Properties properties) throws IOExce
     public void setModuleLogLevel(BLogLevel logLevel, String moduleName) {
         loggerLevels.put(moduleName, logLevel);
     }
+
+    /**
+     * Checks if the output format of logs is set to json.
+     *
+     * @return true if the output format is json, false otherwise
+     */
+    public static boolean isJsonFormat() {
+        return jsonFormat;
+    }
+
+    /**
+     * Sets the output format of logs to json.
+     */
+    public static void setBallerinaJsonLogFormat() {
+        BLogManager.jsonFormat = true;
+    }
```

Comment:
Shall we change the API as
```
public static void setLogOutputFormat(string outputFormat) {
     BLogManager.outputFormat = outputFormat;
}
```

Reply:
And define constants for each supported format.

Reply:
Updated in bc3a3f6